plugins/module_utils/proxmox.py

```diff
@@ -137,3 +137,9 @@ def get_vm(self, vmid, ignore_missing=False):
                 return None
 
             self.module.fail_json(msg='VM with vmid %s does not exist in cluster' % vmid)
+
+    def api_task_ok(self, node, taskid):
+        status = self.proxmox_api.nodes(node).tasks(taskid).status.get()
+        if status['status'] == 'stopped' and status['exitstatus'] == 'OK':
+            return True
+        return False
```

Comment:
Why not simply:
```suggestion
        return status['status'] == 'stopped' and status['exitstatus'] == 'OK'
```
?

Reply:
very very nice :+1: